BackEndLib/Browser.cpp

```diff
@@ -167,6 +167,7 @@ static bool OpenSystemBrowser(
 
 #elif defined(__APPLE__)
 #include <string>
+#include <cstring>
```

Comment:
Why this change?

Reply:
Because `strlen` used in this OSX specific function is defined in `cstring` so we need to include this in addition to `string`.